crates/sys/src/lfrfid.rs

```diff
@@ -0,0 +1,50 @@
+//! Low-level bindings to the LF-RFID API.
+
+use core::ffi::c_char;
+
+use crate::opaque;
+
+
+
+// The global array of all protocols implemented by the firmware.
+opaque!(ProtocolsArray);
+
+// The number of entries in LFRFID_PROTOCOLS
+pub const PROTOCOLS_MAX: usize = 10; // TODO: find some way to keep sync with LFRFIDProtocolMax
```

Comment:
It should be fine if this doesn't match `flipperzero-firmware`.

If `PROTOCOLS_MAX < LFRFIDProtocolMax`, this just means we can't access the new protocols.
If `PROTOCOLS_MAX > LFRFIDProtocolMax` (old firmware), programs will still work, but will [trigger a `furi_crash` if the application tries to use a too-new protocol](https://sourcegraph.com/github.com/flipperdevices/flipperzero-firmware@0f9ea92/-/blob/lib/toolbox/protocols/protocol_dict.c?L76).

If it becomes particularly problematic, we can probably request a `extern` variable with the correct `LFRFIDProtocolMax` value.

Reply:
Sounds good. Hey @hedger any chance we could get `LFRFIDProtocolMax` as an API symbol? ^^'